### PR TITLE
chore(logs): Fix typo

### PR DESCRIPTION
**Pull request by deepyaman**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-logs/lib/log-group.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ abstract class LogGroupBase extends Resource implements ILogGroup {
   }
 
   /**
-   * Give permissions to write to create and write to streams in this log group
+   * Give permissions to create and write to streams in this log group
```
Comment (deepyaman) [inline packages/@aws-cdk/aws-logs/lib/log-group.ts:157]:
```suggestion
   * Give permissions to create and write to streams in this log group 
```
Fix PR semantics

Comment (deepyaman) [inline packages/@aws-cdk/aws-logs/lib/log-group.ts]:
```suggestion
   * Give permissions to create and write to streams in this log group
```

File: packages/@aws-cdk/aws-logs/lib/log-group.ts [hunk 2]
```diff
@@ -154,7 +154,7 @@ abstract class LogGroupBase extends Resource implements ILogGroup {
...
```
Comment (deepyaman) [inline packages/@aws-cdk/aws-logs/lib/log-group.ts:157]:
```suggestion
   * Give permissions to create and write to streams in this log group
```
Fix PR semantics

**Review by SomayaB — APPROVED** [review]
Good catch! Thanks 👍

**Pull request merged**